### ISSUE TITLE
Bulk download all available clips that are associated with one vod

### ISSUE THREAD

**Issue by UndeadKitten** [lay295/TwitchDownloader#883]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

Sometimes there are no reuploads to be found for old vods, but the twitch channel has a ton of clips of that stream. 

It would be nice to be able to input the link to a vod and automatically download all available clips of that vod or input the link to a clip, automatically find what vod it's associated with, then download all available clips of that vod.

You could have each clip labeled in chronological order, so with a media player you could watch them back to back in chronological order, or merge them in a video editor in an attempt to partially reconstruct lost old vods.

**Comment by ScrubN** [comment]
To clarify, you want to pass in the ID to a VOD that is currently available in the user's broadcasts section and find and download clips from that video?

**Comment by UndeadKitten** [comment]
> To clarify, you want to pass in the ID to a VOD that is currently available in the user's broadcasts section and find and download clips from that video?

Yes, the other stuff I mentioned would be for ease of use that could maybe be added later but yes, what you describe is the function that I'd like.

**Comment by ScrubN** [comment]
I recently discovered that the Twitch API actually has a function specifically for getting clips of a specific video, which reminded me of this feature request. There are some deal breakers though: the clips are ordered by their creation date, and finding clips of a deleted video is not supported.
